vantage6-backend-common/vantage6/backend/common/base.py

```diff
@@ -87,6 +88,9 @@ def _connect(
         """
         self.allow_drop_all = allow_drop_all
         self.URI = uri
+        # Number of attempts and delay in second for the db connection
```

Comment:
Should be fine like this, but if this were Spring Boot I would make these variables optionally configurable from the config file. Not sure if it's as straightforward to do that in this case. 
For readability of the code I prefer things like MAX_NUMBER_OF_ATTEMPTS, RETRY_DELAY_IN_SECONDS as opposed to adding comments.

Reply:
Yeah I thought the same.